src/labs/deep-agents.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Deep Agents
+description: Create sophisticated AI agents that can plan, use subagents, and work with files for complex tasks.
+---
+
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs'
+
+# 🧠🤖 Deep Agents
+
+Using an LLM to call tools in a loop is the simplest form of an agent. This architecture, however, can yield agents that are "shallow" and fail to plan and act over longer, more complex tasks. Applications like "Deep Research", "Manus", and "Claude Code" have gotten around this limitation by implementing a combination of four things: a **planning tool**, **sub agents**, access to a **file system**, and a **detailed prompt**.
+
+Deep Agents implements these components in a general purpose way so that you can easily create sophisticated agents for your application, available in both **Python** and **JavaScript/TypeScript**.
+
+## Key Features
+
+- **Planning Tool**: Built-in planning capabilities to break down complex tasks
+- **Sub Agents**: Specialized agents for specific tasks with context quarantine  
+- **Virtual File System**: Mock file system for persistent state without conflicts
+- **Detailed System Prompt**: Heavily inspired by Claude Code's effective prompting
+
+## Installation
+
+<Tabs defaultValue="python">
```

Comment:
These aren't rendering-- you can use Mintlify's "CodeGroup" component for this: https://mintlify.com/docs/components/code-groups#creating-code-groups